### PR TITLE
pass linked interactions to init -- FV_48

### PR DESCRIPTION
**Pull request by ede0m**

### CODE REVIEW

File: floodviz/static/js/hydrograph.js [hunk 1]
```diff
@@ -182,7 +180,8 @@
 		 * Initialize the Hydrograph
 		 */
 		self.init = function (linked_interactions) {
```
Comment (mbucknell) [inline floodviz/static/js/hydrograph.js]: Move the comments describing the functions from the hydromodule comments at the top to here.

File: floodviz/static/js/floodviz_onready.js [hunk 2]
```diff
@@ -26,20 +26,23 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	var hydro_figure = FV.hydromodule(hydro_options);
 
 	// Use frames to link interactions
-	hydro_options['hover_in'] = map_figure.site_tooltip_show;
-	hydro_options['hover_out'] = map_figure.site_tooltip_remove;
-	hydro_options['click_on'] = map_figure.site_add_accent;
```
Comment (mbucknell) [inline floodviz/static/js/floodviz_onready.js]: Where do we use click_on? Shouldn't there just be a click interaction? Maybe we should talk about this.

Comment (ede0m) [inline floodviz/static/js/floodviz_onready.js]: Whoops! Forgot to remove that one when I removed the site_add_accent to map.

File: floodviz/static/js/floodviz_onready.js [hunk 3]
```diff
@@ -26,20 +26,22 @@ document.addEventListener("DOMContentLoaded", function (event) {
...
-	hydro_options['click_off'] = map_figure.site_remove_accent;
+	var map_to_hydro = {
+		'hover_in': map_figure.site_tooltip_show,
+		'hover_out': map_figure.site_tooltip_remove,
+		'click_off': map_figure.site_remove_accent,
```
Comment (mbucknell) [inline floodviz/static/js/floodviz_onready.js]: Can we call this 'click'? That's the event that the action is linked to

File: floodviz/static/js/floodviz_onready.js [hunk 4]
```diff
@@ -26,20 +26,22 @@ document.addEventListener("DOMContentLoaded", function (event) {
...
+	}
+	var hydro_to_map = {
+		'hover_in': hydro_figure.activate_line,
+		'hover_out': hydro_figure.deactivate_line,
+		'click_toggle' : hydro_figure.change_lines
```
Comment (mbucknell) [inline floodviz/static/js/floodviz_onready.js]: Can we call this one 'click'? That's the event that the action is linked to.

**Pull request merged**